### PR TITLE
Refactor: sort_canisters backend

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Refactor the sort canisters function for a method.

# Changes

* Add `sort` method to `canisters` field in `AccountsStore`.
* Substitut calls to `sort_canisters` for new method.

# Tests

* No changes in functionality, just in implementation.

# Todos

Nothing changed for the user, it's a code style change.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@bitdivine please review

**Review by bitdivine — APPROVED** [review]
Thank you.

File: rs/backend/src/accounts_store.rs [hunk 1]
```diff
@@ -796,7 +825,7 @@ impl AccountsStore {
                     name: request.name,
                     canister_id: request.canister_id,
                 });
-                sort_canisters(&mut account.canisters);
+                account.canisters.sort();
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply rs/backend/src/accounts_store.rs:828]: You still need to remove it right? Insert shifts all the other elements:

https://doc.rust-lang.org/std/vec/struct.Vec.html#method.insert

> Inserts an element at position index within the vector, shifting all elements after it to the right.

[the remark this answers is not in the document]
Comment (bitdivine) [reply rs/backend/src/accounts_store.rs:828]: Yes, inserting has to move.  But so does sorting if a new element has been appended to the end and the sort then has to move the element to somewhere in the middle of the array.  But this really is nit-picking and doesn't matter given the small lists we typically have to worry about.  I really am 100% happy with merging as is.  If and when performance becomes a problem we can spend time on it.

[the remark this answers is not in the document]
Comment (lmuntaner) [reply rs/backend/src/accounts_store.rs:828]: Sounds good to me!

**Pull request merged**